integration/chaincode/gateway/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/hyperledger/fabric/integration/chaincode/gateway
+
+go 1.15
```

Comment:
We're not using go 1.15 in Fabric yet.

Reply:
It looks like #2386 has updated Fabric to use go 1.15

Reply:
Yes, but most modules still have 1.13 or 1.14 compatibility levels declared in them because we're not making use of go 1.15. This significance of this directive is growing now that modules are the "standard" way of dealing with dependencies.

https://golang.org/ref/mod#go-mod-file-go

That said, I've already indicated in other comments that I don't think this chaincode (and associated module) is needed to test what you're trying to test and we should use or adapt what we have.